data/transactions/logic/TEAL_opcodes.md

```diff
@@ -1104,6 +1122,15 @@ Fails unless the last instruction executed was a `callsub`.
 - branch to the Ath label. Continue at following instruction if index A exceeds the number of labels.
 - Availability: v8
 
+## match target ...
+
+- Opcode: 0x8e {uint8 branch count} [{int16 branch offset, big-endian}, ...]
+- Stack: ..., [A1, A2, ..., AN], B &rarr; ...
+- given match cases from A[0] to A[N-1], branch to the Ith label where A[I] = B. Continue to the following instruction if no matches are found.
```

Comment:
Still some disagreement on whether A is 0 based or 1 based. I vote for 1 based, only so the Stack notation can avoid using "N-1" as a "subscript".